docs/source/install.rst

```diff
@@ -113,65 +111,70 @@ When your system meets the minimum required kernel release, you can use the foll
+    # sudo docker run hello-world
 
 This command displays output similar to the following truncated example. The first section of the output contains a message about the installation status::
 
-  Unable to find image 'hello-world:latest' locally
-  latest: Pulling from hello-world
-  a8219747be10: Pull complete
-  91c95931e552: Already exists
-  hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
-  Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd18681cf5daeb82aab55838d
-  Status: Downloaded newer image for hello-world:latest
-  Hello from Docker.
-  This message shows that your installation appears to be working correctly.
+    Unable to find image 'hello-world:latest' locally
+    latest: Pulling from hello-world
+    a8219747be10: Pull complete
+    91c95931e552: Already exists
+    hello-world:latest: The image you are pulling has been verified. Important: image verification is a tech preview feature and should not be relied on to provide security.
+    Digest: sha256:aa03e5d0d5553b4c3473e89c8619cf79df368babd18681cf5daeb82aab55838d
+    Status: Downloaded newer image for hello-world:latest
+    Hello from Docker.
+    This message shows that your installation appears to be working correctly.
 
-  ...
+    ...
 
 
-3. To ensure that Docker starts when you start your system, enter the following command:
+3. To ensure that Docker starts when you start your system, enter the following command::
 
-  ``# sudo chkconfig docker on``
+    # sudo chkconfig docker on
 
 After you complete the steps to install Docker for Red Hat Enterprise Linux 6.6 or later, you can continue with the steps to obtain the Quipucords server container image.
 
 Installing the Quipucords Server Container Image
 ------------------------------------------------
 After Docker is installed, you can obtain and install the container image that enables the use of the Quipucords server.
 
-Start by downloading the server container image from the provided URL::
+1. Download the server container image by entering the following command::
+
+    #  curl -k -O -sSL https://ftp.redhat.com/repo/container/quipucords.1.0.0.tar.gz
+
+
+2. Load the container image into the local Docker registry with the following command::
+
+    #  sudo docker load -i quipucords.1.0.0.tar.gz
 
-  #  curl -k -O -sSL https://ftp.redhat.com/repo/container/quipucords.1.0.0.tar.gz
+The output appears similar to the following example::
 
+    Loaded image: quipucords:1.0.0
 
-Load the container image into the local Docker registry with the following command::
 
-  #  sudo docker load -i quipucords.1.0.0.tar.gz
-  ...
-  Loaded image: quipucords:1.0.0
+3. Verify the image within the local Docker registry by entering the following command::
 
+    #  sudo docker images
 
-You can verify the image within the local Docker registry::
+The output appears similar to the following example::
```

Comment:
Earlier in the page, we say "the output of this command is similar to the following example::" (line 104) Do we want to have consistent language, or is it okay that it varies a little?

Reply:
That is a good note. I'll try and get this updated in the future.